commcare_connect/opportunity/views.py

```
@@ -3781,7 +3781,8 @@ def create_task(request, org_slug, opp_id):
         )
     except TaskAlreadyAssignedError:
         messages.error(request, _("This task type is already assigned to the selected worker."))
-    except CommCareHQAPIException:
+    except CommCareHQAPIException as e:
+        logger.exception(f"CommCareHQ task creation failed: {str(e)}")
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)